codalab/lib/worksheet_util.py

```diff
@@ -443,7 +443,9 @@ def interpret_file_genpath(client, target_cache, bundle_uuid, genpath, post):
                 contents = client.head_target(target, MAX_LINES)
                 contents = map(base64.b64decode, contents)
 
-            if all('\t' in x for x in contents):
+            if not contents: # contents is empty list
+              info = ''
```

Comment:
You should make your editor default to 4 for Python.   Make sure you don't add tabs.  Indentation is really important to get right for Python!